packages/git_lfs.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Git_lfs < Package
+  description 'Git extension for versioning large files'
+  homepage 'https://git-lfs.github.com/'
+  version '2.2.1'
+  source_url 'https://github.com/git-lfs/git-lfs/archive/v2.2.1.tar.gz'
+  source_sha256 'fede2b31b0539fd4a580f831867caac1b5d5dc4405e938c4ee0bfeacfb78ad7a'
+
+  depends_on 'go'
+
+  def self.build
```

Comment:
Don't you mind to put `ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"` at the beginning of build function to save ppl who doesn't set TMPDIR correctly like what `kubectl.rb` did?